### PR TITLE
no more removing reagents (from living mobs) with droppers

### LINKED ISSUE #11086: The baster can dupe chems
(WEB REPORT BY: gutlesscoward REMOTE: 172.93.109.202:7777)

> Revision 2e470598b259fb4d1576c1b9f99068ace1139f54 
> General description of the issue
> you can remove/add chemicals to people via the baster; if you keep removing and adding a chemical from a person fast enough with the baster, you can end up with more chemicals than you initially put in.
> 
> What you expected to happen
> I expected to end up with the same amount (or less, due to metabolism) amount of chemicals
> What actually happened
> I was able to  take out more of a chemical from myself than I had put in
> 
> Steps to reproduce if possible
> 1. Get Baster
> 2. Fill it with some chemical
> 3. click rapidly on a human to transfer the chemical, take it out of them, and squit it back in them. make sure the human isn't wearing eye protection. there will be no message for removing the chemical from the human
> 4. if done correctly, you will be putting more into the human than you are taking out. it's easily noticable if you only put in 1 unit of something into a human, because the transfer message will show that, as you're clicking on the human, you're putting in more and more chemical. you could also check via an advanced mass-spectrometer.

additionally, you can remove the reagents from the human with the baster and put them into beakers for later use.

### PR DESCRIPTION
**Pull request by coldcola**
fixes #11086
dropper

### CODE REVIEW

**Comment by jakmak** [comment]
how are we suppose to preform ghetto dialyses now?

**Comment by coldcola** [comment]
truly it is a sad bug to see go, but the temptation to 'abuse' is so strong it must be done

File: code/modules/reagents/reagent_containers/dropper.dm [hunk 1]
```diff
@@ -83,7 +83,9 @@
 		if(!target.is_open_container() && !istype(target,/obj/structure/reagent_dispensers))
```
[the remark this answers is not in the document]
Comment (coldcola) [reply code/modules/reagents/reagent_containers/dropper.dm:83]: this reverts bug

File: code/modules/reagents/reagent_containers/dropper.dm [hunk 2]
```diff
@@ -83,7 +83,9 @@
...
 			to_chat(user, "<span class='warning'>You cannot directly remove reagents from [target].</span>")
 			return
-
+		if(target.is_open_container() && istype(target,/mob/living/carbon))
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/dropper.dm]: ismob

**Pull request merged**